Assets/HoloToolkit/Sharing/Tests/ImportExportAnchorManager.cs

```diff
@@ -500,7 +505,7 @@ void Export()
     /// Called by the WorldAnchorTransferBatch as anchor data is available.
     /// </summary>
     /// <param name="data"></param>
-    public void WriteBuffer(byte[] data)
```

Comment:
These should probably stay public, if they were already public. I just meant the things that weren't explicit.